packages/subgraph/schema.graphql

```diff
@@ -59,6 +63,24 @@ type ERC721Balance implements TokenBalance @entity {
   lastUpdated: BigInt!
 }
 
+type ERC1155Balance implements TokenBalance @entity {
+  id: ID!
+  token: ERC1155Contract!
+  dao: Dao!
+  balances: [ERC1155TokenIdBalance!]!
+  lastUpdated: BigInt!
+}
+
+type ERC1155TokenIdBalance @entity {
+  id: ID!
+  token: ERC1155Contract!
+  tokenId: BigInt!
+  amount: BigInt!
+  metadataUri: String!
+  dao: Dao!
+  lastUpdated: BigInt!
+}
```

Comment:
is this entity is for many-to-many or one-to-many ?
perhaps it needes a comment explaining,
and why there is an "id" in the name?